apps/flame-defi/app/config/hooks/useConfig.ts

```diff
@@ -21,5 +22,20 @@ export const useEvmChainData = () => {
   const evmChainsData = Object.values(evmChains);
   const selectedChain = evmChainsData[0] as EvmChainInfo;
 
-  return { selectedChain };
+  const evmChainConfig: Chain = {
```

Comment:
Remove in favor of using `evmChainToRainbowKitChain(selectedChain) as Chain` for now